lib/autobuild/import/archive.rb

```diff
@@ -183,6 +183,9 @@ def update_cache(package)
                         if retries = self.retries
                             additional_options << "--tries" << retries
                         end
+			if Autobuild.bsd?
+                            additional_options << "--no-check-certificate"
```

Comment:
I updated the PR and changed to CURL for BSD like systems (including Macos -> Darwin is a BSD, we workarounded in the past)

For the log: Please find attached the output for wget:
```
[dev@freebsd-test /tmp]$ wget rock-robotics.org/master/autoproj_bootstrap
--2015-03-24 08:14:25--  http://rock-robotics.org/master/autoproj_bootstrap
Resolving rock-robotics.org (rock-robotics.org)... 46.252.18.41, 2a00:1158:0:300:abe6::1
Connecting to rock-robotics.org (rock-robotics.org)|46.252.18.41|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://raw.githubusercontent.com/rock-core/autoproj/stable/bin/autoproj_bootstrap [following]
--2015-03-24 08:14:25--  https://raw.githubusercontent.com/rock-core/autoproj/stable/bin/autoproj_bootstrap
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.31.17.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.31.17.133|:443... connected.
ERROR: cannot verify raw.githubusercontent.com's certificate, issued by 'CN=DigiCert SHA2 High Assurance Server CA,OU=www.digicert.com,O=DigiCert Inc,C=US':
  Unable to locally verify the issuer's authority.
To connect to raw.githubusercontent.com insecurely, use `--no-check-certificate'.
```